ckanext/datastore/db.py

```diff
@@ -1173,6 +1195,37 @@ def create(context, data_dict):
         context['connection'].close()
 
 
+def _create_triggers(connection, resource_id, triggers):
+    u'''
+    Delete existing triggers on table then create triggers
+
+    Currently our schema requires "before insert or update"
+    triggers run on each row, so we're not reading "when"
+    or "for_each" parameters from triggers list.
+    '''
+    existing = connection.execute(
+        u'SELECT tgname FROM pg_trigger WHERE tgrelid = %s::regclass',
```

Comment:
the execute() call is quoting the resource_id parameter for us. I use `identifier` when I can't avoid hand-rolling sql.